prog/main.go

```diff
@@ -119,6 +121,25 @@ type appFlags struct {
 	consulInf       string
 }
 
+func logCensoredArgs() {
+	var prettyPrintedArgs string
+	// we show the args followed by the flags which is likely to change the
```

Comment:
I did it this way because we use a launch subcommand. But ... in hindsight it doesn't help because the launch command is translated into a `--mode` flag anyways. I will change it.